### PR TITLE
feat: add repo delete and bump deps

### LINKED ISSUE #321: Drop the repo from the dashboard or API
After I create a "test" repo in the central.artipie, I realize that I can't drop/delete the repo from the site.  We need this feature and the api.

### PR DESCRIPTION
**Pull request by genryxy**
Closes #321 
Implemented repo delete and bumped `asto` to `v1.11.0` and `api` to `v0.5.1`.
Get name without extension in `RepositoriesFromStorage`

### CODE REVIEW

File: src/main/java/com/artipie/MeasuredStorage.java [hunk 1]
```diff
@@ -169,14 +170,8 @@ public void onComplete() {
...
-    public CompletableFuture<Long> size(final Key key) {
-        final long start = System.nanoTime();
-        return this.origin.size(key).thenApply(
-            res -> {
-                this.log("size(%s): %s", key.string(), millisMessage(start));
-                return res;
-            }
-        );
+    public CompletableFuture<? extends Meta> metadata(final Key key) {
+        return this.origin.metadata(key);
```
Comment (olenagerasimova) [inline src/main/java/com/artipie/MeasuredStorage.java:174]: Every method of `MeasuredStorage` logs execution time, I'd suggest logging it here too

**Review by olenagerasimova — CHANGES_REQUESTED** [review]
@genryxy please, check my comment

**Comment by genryxy** [comment]
@olenagerasimova thanks, corrected

**Review by olenagerasimova — APPROVED** [review]
@genryxy thanks, looks good

**Comment by genryxy** [comment]
@olenagerasimova I've disabled conda install IT case because it fails with out of storage [exception](https://github.com/artipie/artipie/runs/4641400517?check_suite_focus=true#step:6:2241) after bumping asto to `v1.11.0`

**Comment by genryxy** [comment]
@g4s8 could you please check / merge

**Comment by genryxy** [comment]
@g4s8 could you please check

**Review by g4s8 — APPROVED** [review]
@genryxy thanks

**Pull request merged**